.prettierrc

```diff
@@ -2,5 +2,4 @@
   "semi": true,
   "singleQuote": true,
   "printWidth": 100,
-  "parser": "typescript"
```

Comment:
Why is this removed? As far as I'm concerned, the prettier only looks at typescript files due to another config, and not this config.

Reply:
I wanted to prettify scss files with prettier. It's redundant either way. prettier knows what parser to use. I reverted it nevertheless.